ParkingLot/ParkingBoy.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ParkingLot
+{
+    public class ParkingBoy
+    {
+        private List<ParkingLot> boysParkingLots = new List<ParkingLot>();
+        public ParkingBoy(string id)
+        {
+            this.Id = id;
+        }
+
+        public string Id { get; set; }
+
+        public List<ParkingLot> BoysParkingLots => boysParkingLots;
+
+        public void AddParkingLot(ParkingLot newParkingLot)
+        {
+            this.boysParkingLots.Add(newParkingLot);
+        }
+
+        public Ticket ParkCarBoy(Car car, string customerId)
+        {
+            Ticket newTicket = null;
+            if (car != null && car.IsParked == false)
+            {
+                ParkingLot parkingLot = FindAvaibleParkingLot();
+                bool isParkSuccess = parkingLot.ParkCarLot(car);
+                if (isParkSuccess)
+                {
+                    newTicket = new Ticket(parkingLot.Id, this.Id, car.Id, customerId);
+                    car.IsParked = true;
```

Comment:
Have parking boy id may lead to problem for manager having multiple parking boy and every parking boy can park/fetch the car